### DISCUSSION TITLE
Android TV

### DISCUSSION THREAD

**Discussion by fairking** [TeamNewPipe/NewPipe#10843] [Questions]
Could not use NewPipe on Android TV (GT King, Android 10) :-(
Error happens when switching to fullscreen.

**Comment by fairking** [comment]
![image](https://github.com/TeamNewPipe/NewPipe/assets/13495631/eeae52b0-d024-4b25-bb88-7271f3cd1813)

![image](https://github.com/TeamNewPipe/NewPipe/assets/13495631/ffac0d90-94bd-4ea8-85c3-f35054ab4615)

**Comment by opusforlife2** [comment]
Discussions are for questions, not bugs. Open a bug report issue.
